lib/runner/test-runners/default.js

```diff
@@ -152,13 +161,17 @@ class DefaultRunner {
   }
 
   runConcurrent(testEnvArray, modules) {
-    const Concurrency = require('../concurrency/concurrency.js');
-
     this.concurrency = new Concurrency(this.settings, this.argvOpts);
     this.globalReporter.setupChildProcessListener(this.concurrency);
 
-    return this.concurrency.runMultiple(testEnvArray, modules);
+    return this.concurrency
+      .runMultiple(testEnvArray, modules)
+      .then(exitCode => {
+        this.reportResults();
```

Comment:
`this.reportResults()` returns a Promise so we should wait for it.
```suggestion
        return this.reportResults().then(hasFailures => {
          return exitCode
        })
```